wgsl/index.bs

```diff
@@ -3516,8 +3516,8 @@ the test name.
 * v-0002: Non-void functions must end with a return statement.
 * v-0003: At least one of vertex, fragment or compute shader must be present.
 * v-0004: Recursion is not allowed.
-* v-0005: Functions must be declared before use.
-* v-0006: Variables must be defined before use.
+* v-0005: Types must be declared before use.
```

Comment:
isn't type name an identifier? if so, would the second sentence be a generalization of the first?

Reply:
That's right. How about I use the rule From https://gpuweb.github.io/gpuweb/wgsl.html#scoping to take all the name uniqueness rules and condense them to one?